addons/stock/wizard/product_replenish.py

```diff
@@ -100,16 +111,44 @@ def launch_replenishment(self):
         except UserError as error:
             raise UserError(error)
 
-    def _prepare_orderpoint_values(self):
```

Comment:
Keep the method in stable. We will remove it in master. You can still add your own below